test/ClangImporter/indirect_field_codegen.swift

```diff
@@ -0,0 +1,22 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift -I %S/Inputs/custom-modules %s -o %t/a.out
```

Comment:
Can you make this an `-emit-silgen` test, so we can see what names it picked for each of them? `%target-swift-frontend -emit-silgen … | %FileCheck %s`, and then CHECK lines for the heads of the SIL functions we expect to be there.

Reply:
(Pushed with changes suggested in Slack.)